docs/DeveloperGuide.md

```diff
@@ -436,121 +441,107 @@ All get commands are implemented in the following steps:
+Getting the list of patients who are taking the query long term medication involves the following steps:
+1. Prefix `/m` is matched in `GetCommandParser` class
+2. A new `GetMedicationCommandParser` instance is created and parses the user input
+3. A `GetMedicationCommand` instance is returned
+4. The model is updated such that the *filtered* list only displays patients who are taking the query long term medication
 
-Details of the next of kin include the name, relationship to patient and phone number.
+Medication name searches are case-insensitive. Each search requires a full-word match.
+For example, searching for `paracet` will only return patients with long-term medication name
+`paracet`, and **not** `paracetamol`.
 
-#### Ward Number (/wn)
+#### Name (/n)
 
-Getting the list of patients in the query ward number involves the following steps:
-1. prefix "/wn" is matched in `GetCommandParser` class
-2. a new `GetWardNumberCommandParser` instance is created and parses the user input
-3. a `GetWardNumberCommand` instance is returned
-4. the model is updated such that the *filtered* list only displays patients who are in the query ward number
+Getting the list of patients whose name contains the query name involves the following steps:
+1. Prefix `/n` is matched in `GetCommandParser` class
+2. A new `GetNameCommandParser` instance is created and parses the user input
+3. A `GetNameCommand` instance is returned
+4. The model is updated such that the *filtered* list only displays patients whose name contains the query name
 
-Strict restrictions are placed to prevent too many varieties of ward number inputs. This way the regex for searching
-for ward numbers is simplified. Due to differing places having different ways of numbering their ward numbers, we
-have standardised it to be in the format of `Uppercase Alphabet` + `3 Numbers`. For example, `A123`, `B241`, `C005`, etc.
+Name searches are case-insensitive. Each search requires a full-word match.
+For example, searching for `john` will only return patients with name `John`, and **not** `Johnson`.
 
-#### Appointment Date (/appton)
+#### Next of Kin (/nok)
 
-Getting the list of patients in th query appointment date involves the following steps:
-1. prefix "/appton" is matched in `GetCommandParser` class
-2. a new `GetAppointmentByDateCommandParser` instance is created and parsers the user input
-3. a `GetAppointmentByDateCommand` instance is returned
-4. the model is updated such that the *filtered* list only displays patients who have the same query appointment date
+Getting the information of the next of kin of the list of query patient names involves the following steps:
+1. Prefix `/nok` is matched in `GetCommandParser` class
+2. A new `GetNextOfKinCommandParser` instance is created and parses the user input
+3. A `GetNextOfKinCommand` instance is returned
+4. The model is updated such that the *filtered* list only displays queried patients' next of kin details
```

Comment:
I think there's sth strange abt the tense, everything on top uses query(which Im not sure if it is the correct tense). Best to standardise the tense bah

Reply:
Got it, will standardise tense used across the `get` commands.